internal/pkg/manifest/worker_svc.go

```diff
@@ -125,16 +111,66 @@ func (q *SQSQueueOrBool) UnmarshalYAML(value *yaml.Node) error {
 
 // SQSQueue represents the configurable options for setting up a SQS Queue.
 type SQSQueue struct {
-	Retention  *time.Duration  `yaml:"retention"`
-	Delay      *time.Duration  `yaml:"delay"`
-	Timeout    *time.Duration  `yaml:"timeout"`
-	DeadLetter DeadLetterQueue `yaml:"dead_letter"`
+	Retention  *time.Duration          `yaml:"retention"`
+	Delay      *time.Duration          `yaml:"delay"`
+	Timeout    *time.Duration          `yaml:"timeout"`
+	DeadLetter DeadLetterQueue         `yaml:"dead_letter"`
+	FIFO       FIFOAdvanceConfigOrBool `yaml:"fifo"`
+}
+
+// FIFOAdvanceConfigOrBool represents the configurable options for fifo queues.
+type FIFOAdvanceConfigOrBool struct {
+	Enable   *bool
+	Advanced FIFOAdvanceConfig
+}
+
+// IsEmpty returns true if the FifoAdvanceConfigOrBool struct has all nil values.
+func (f *FIFOAdvanceConfigOrBool) IsEmpty() bool {
+	return f.Enable == nil && f.Advanced.IsEmpty()
+}
+
+// IsEmpty returns true if the FifoAdvanceConfigOrBool struct has all nil values.
+func (f *FIFOAdvanceConfigOrBool) IsEnabled() bool {
+	return aws.BoolValue(f.Enable) || !f.Advanced.IsEmpty()
+}
+
+// FifoAdvanceConfig represents the advanced fifo queue config.
+type FIFOAdvanceConfig struct {
+	ContentBasedDeduplication *bool   `yaml:"content_based_deduplication"`
+	DeduplicationScope        *string `yaml:"deduplication_scope"`
+	FIFOThroughputLimit       *string `yaml:"fifo_throughput_limit"`
+	HighThroughputFifo        *bool   `yaml:"high_throughput_fifo"`
```

Comment:
can we rename this field to remove the "_fifo" suffix? it's redundant :) 
```suggestion
	HighThroughputFifo        *bool   `yaml:"high_throughput"`
```